### PR TITLE
Fix discrepencies in Hour of Code event counts

### PR DESCRIPTION
**Pull request by TurnerRiley**
Currrently, the Hour of Code country-specific event pages (e.g. [for Spain](https://hourofcode.com/es)) show different event counts than the [main list of all Hour of Code events](https://hourofcode.com/us/events/all):
WRONG_NUM
Spain_total

The event pages do [their own counting](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/cron/analyze_hoc_activity#L97-L100) of the events whereas the main events list page [uses the helper function `events_by_country`](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/helper_modules/forms.rb#L27). To unify them, I changed the event pages to use the same helper function.

Now, the counts are consistent (since I can't view the hour of code page on an adhoc, I can only view the cronjob output that I printed):
EVENT_TOTALS_COUNTRY
Spain_total

## Links
Jira ticket: [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?assignee=60d62161f65054006980bd71&selectedIssue=ACQ-1178)

## Testing story
Set up an adhoc, ran `./bin/cron/analyze_hoc_activity`, and printed out the resulting country event counts to compare. Since I can't see the Hour of Code pages on an adhoc, I can only view the output from the `analyze_hoc_activity` script (I temporarily added some `puts` statements).

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/yourschool.haml [hunk 1]
```diff
@@ -52,6 +55,9 @@ social:
 
 #partners
   %hr
+
+  %p= @hoc_data
```
Comment (hannahbergam) [inline pegasus/sites.v3/code.org/public/yourschool.haml]: Is this replacing something else? I'm not quite sure I'm following how the yourschool file comes into play

**Review by hannahbergam — COMMENTED** [review]
A quick question! Otherwise the testing part makes sense

File: pegasus/sites.v3/code.org/public/yourschool.haml (hunk 1 above)
Comment (TurnerRiley) [reply pegasus/sites.v3/code.org/public/yourschool.haml]: Ah sorry, I still had one more commit to push where I remove these changes (this was for showing the output in a non-hoc site location)

**Review by hannahbergam — APPROVED** [review]
Thanks for the update! LGTM!

**Pull request merged**